docs/development/intellij.md

```diff
@@ -0,0 +1,10 @@
+# Code style settings
+
+## Java
+
+1. Open your Intellij Idea settings
+2. Navigate to Editor -> Code Style -> Java
+3. From Scheme options select Import Scheme -> Eclipse XML Profile
+![code_style_settings.png](./attachments/images/code_style_settings.png)
+4. Open `./config/Dimpact.xml` from the project root
```

Comment:
Oh, yes I would not name this `Dimpact.xml`. This open source project should really stand on it's own and be independent of 'Dimpact' I would rename this just `zac.xml`.